### ISSUE TITLE
Rich Text to Markdown converter doesn't work (anymore)

### ISSUE THREAD

**Issue by brianphillips** [drodil/backstage-plugin-toolbox#65]
I recently submitted a PR that was merged (#58) but it appears that the contents of the change were almost entirely ripped out in a subsequent commit  (see [here](https://github.com/drodil/backstage-plugin-toolbox/commit/4425521b11d61a9c83f87979c89a18630ef6dd5d#diff-22a09ef64d4d1647612c997db2336d97772895d402a2eb6bc63d9eda02957c34)). I'm wondering if this was inadvertant (probably?) or if I put things back together, if there's something I need to take into account to avoid breaking something else or from having someone undo the work I did.

Just wanted to check before I reverted what was done to that specific tool.

Thanks!

**Comment by drodil** [comment]
Hi,

Hmm, yes it might be that it broke, though I tried to go through the functionalities. I was refactoring the tools adding the new download/upload file functionality and also to use the default editor wherever possible. Also I added the preview for the markdown content. What exactly is not working with the tool anymore? At least I can still convert HTML to MD without issues:
![image](https://github.com/drodil/backstage-plugin-toolbox/assets/1178319/a24da2f9-29a6-4742-865c-b513af946d07)

**Comment by brianphillips** [comment]
Good idea, I will see if an option can be passed in to the default editor to modify the "paste" button (or perhaps to ad a "Paste with formatting" button?) so that not so much needs to be customized for this specific tool. Thanks

**Comment by ppiwow-apay** [comment]
any news on that? seems to be really useful tool, specially when i want to convert docs to adr/tech docs, as on collaboration docs are pretty usefull
